client/common/client.c

```diff
@@ -1896,7 +1898,7 @@ static BOOL freerdp_client_register_pen(rdpClientContext* cctx, UINT32 flags, IN
 	}
 
 	size_t pos = 0;
-	FreeRDP_PenDevice* pen = freerdp_client_get_pen(cctx, deviceid, &pos);
+	FreeRDP_PenDevice* pen = freerdp_client_get_pen(cctx, null_deviceid, &pos);
```

Comment:
ok, this is a bug, but the solution does not look correct.
1. should check for existing device with same id -> fail
2. should find first free slot and register there